src/foam/u2/layout/AlignmentTypes.js

```diff
@@ -0,0 +1,24 @@
+foam.ENUM({
+  package: 'foam.u2.layout',
+  name: 'AlignmentTypes',
+
+  documentation: `
+      An enum for the alignment types of a ~row of columns~
+  `,
+
+  values: [
+    { name: "START", label: 'Normal', webFlexProp: 'flex-start' },
```

Comment:
Are these labels right? I wouldn't even add the labels tbh. I don't think they bring in much value in this case.

Reply:
Done